### ISSUE TITLE
[19w28a] Canopies treat hollow insides as physical collision box

### ISSUE THREAD

**Issue by Majikmonster** [tsunamayo/Skywanderers#1326]
### Short Description of the issue:
The underside of the small canopy blocks treat themselves as whole blocks, preventing them from being placed even when their physical appearance suggests there is no collision.

### Steps to reproduce the issue:

- place a single brick on a flat surface
- place a canopy on that surface and attempt to stretch it over top of the brick, where the brick would not collide with the class or frame.

### Environment
- Skywanderers version: 19w28a
- Graphics Card: Nvidia GeForce GTX 1060 6gb

### Additional information:
![20190708084507_1](https://user-images.githubusercontent.com/51390907/60817307-68005100-a169-11e9-9582-97477db821b8.jpg)
![20190708084509_1](https://user-images.githubusercontent.com/51390907/60817316-6afb4180-a169-11e9-93aa-583adfcde70a.jpg)

[output_log.txt](https://github.com/tsunamayo/Skywanderers/files/3368773/output_log.txt)

**Comment by tsunamayo** [comment]
Yes unfortunately it is gonna stay like that until I make progress on the new collision framework... I had to do that because convex / non convex collision detection from unity was bugged... If it is too much of a deal breaker I can try to find a solution on the short term. 
By the way could you send me your ship blueprint? As you are using a lot of brick triangle it seems I want to calibrate the border on brick, they seems too big. Cheers

**Comment by Majikmonster** [comment]
Here is the blueprint:

[2019 june fighter 1 logic bugged_painted.zip](https://github.com/tsunamayo/Skywanderers/files/3371342/2019.june.fighter.1.logic.bugged_painted.zip)

**Comment by tsunamayo** [comment]
It will need to have compound collider to work correctly. I keep you posted.
